cmd/login.go

```diff
@@ -50,17 +52,17 @@ The credential will be stored in your system vault.`, appCtx.PasswordEnvVar()),
 			appCtx, _ := context.AppContext()
 			username := loginFlags.username
 			if username == "" {
-				username = os.Getenv(appCtx.UsernameEnvVar())
-				if username == "" {
-					fmt.Printf("Please enter your user name: ")
-					nb, err := fmt.Scan(&username)
-					if err != nil {
-						return err
-					}
-
-					if nb != 1 {
-						return fmt.Errorf("invalid entries (expected only one argument)")
-					}
+				reader := bufio.NewReader(os.Stdin)
+				defaultUser := defaultUsername()
```

Comment:
I think this will introduce a regression. In the past, we use the UsernameEnvVar() function to get the username from `[PREFIX]_USERNAME` environment variable. I would propose include this in the defaultUsername() function to keep this behavior.

Reply:
Done.

The `defaultUsername()` will select the 1st available option now:
1. `CL_USERNAME`
2. `USER`
3. `USERNAME`